src/gvars.c

```diff
@@ -639,7 +718,7 @@ UInt GVarName (
         SET_ELM_GVAR_LIST( ValGVars,    numGVars, 0 );
         SET_ELM_GVAR_LIST( NameGVars,   numGVars, string );
         CHANGED_GVAR_LIST( NameGVars,   numGVars );
-        SET_ELM_GVAR_LIST( WriteGVars,  numGVars, INTOBJ_INT(1) );
```

Comment:
So if we kept this line (possibly changing the 1 to 0), and leave in the `SetGVarWriteState` in the next line, all should be fine, I guess?